smi-conformance/grpc/handlers.go

```diff
@@ -98,7 +105,7 @@ func (s *Service) RunTest(ctx context.Context, req *conformance.Request) (*confo
 
 			// A hacky way to see the testStep Failed, since KUDO only provides it in Failure.Message
 			re := regexp.MustCompile(`[0-9]+`)
-			if res.Failure != nil {
+			if res.Failure.Message != "" {
```

Comment:
Wt if Failure is `nil`? @dhruv0000

Reply:
This was the reason for failure in `docker build`. 
`invalid operation: res.Failure != nil (mismatched types test_gen.Failure and nil)`

Reply:
yes, do `*res.Failure != nil`